tls/s2n_security_policies.c

```diff
@@ -23,144 +23,172 @@ const struct s2n_security_policy security_policy_20170210 = {
     .minimum_protocol_version = S2N_TLS10,
     .cipher_preferences = &cipher_preferences_20170210,
     .kem_preferences = &kem_preferences_null,
+    .signature_preferences = &s2n_signature_preferences_20140601,
 };
 
 const struct s2n_security_policy security_policy_20190801 = {
     .minimum_protocol_version = S2N_TLS10,
     .cipher_preferences = &cipher_preferences_20190801,
     .kem_preferences = &kem_preferences_null,
+    /* The discrepancy in the date exists because the following signature
+     * preference was added in 20200207 and we donot wish to update the
+     * naming of cipher preferences and security policy associated with it.
+     */
```

Comment:
"donot" -> "don't". Also, I'd be more specific: the signature preferences were named when cipher preferences and signature preferences were tracked separately, and we chose to keep the cipher preference name because customers use it.